README.md

```diff
@@ -59,8 +59,8 @@ This handbook will be for Kuala Lumpur, the capital city of Malaysia, for the ti
 * If you hail a cab, ask them to turn on the meter or use GrabTaxi
 
 ## Food
-* Street food for the win @JalanAlor! https://youtu.be/OzkC8HRjHw0
-* https://www.tourplus.my/en/attractions/foot-hunt-in-jalan-alor
+* Street food for the win @JalanAlor! [https://youtu.be/OzkC8HRjHw0]
+* (Jalan Alor)[https://www.tourplus.my/en/attractions/foot-hunt-in-jalan-alor]
```

Comment:
I think you terbalik the markdown syntax :D

I think its `[`Jalan Alor`]``(`https://www.tourplus.my/en/attractions/foot-hunt-in-jalan-alor`)`